modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-web/src/main/resources/META-INF/resources/admin/process/kaleo_draft_definition_action.jsp

```diff
@@ -38,11 +38,13 @@ KaleoDefinitionVersion kaleoDefinitionVersion = (KaleoDefinitionVersion)row.getO
 		<portlet:param name="draftVersion" value="<%= kaleoDefinitionVersion.getVersion() %>" />
 	</liferay-portlet:renderURL>
 
-	<liferay-ui:icon
-		message="edit"
-		onClick='<%= "javascript:" + liferayPortletResponse.getNamespace() + "editWorkflow('" + editURL + "');" %>'
-		url="javascript:;"
-	/>
+	<c:if test="<%= permissionChecker.isCompanyAdmin() %>">
+		<liferay-ui:icon
```

Comment:
Hey @MariMalta, please move the `liferay-portlet:renderURL` declaration to the if statement as well.

Thx